src/sagas/items.js

```diff
@@ -0,0 +1,40 @@
+import { call, takeEvery, put } from 'redux-saga/effects';
+import {
+  getTrending,
+  getSearch
+} from '../api';
+import {
+  getItemsSuccess,
+  getItemsError
+} from '../actions';
+import {
+  GET_TRENDING_ITEMS,
+  GET_SEARCH_ITEMS
+} from '../actions/types';
+
+export function* loadTrendingItems() {
```

Comment:
Do you notice a pattern here? This is ok for smaller projects like this one, but how would you improve this saga creation to avoid repeating almost the same code for each saga?